### PR TITLE
GAWB-2500 run tests against broad-dsde-dev

### PR DESCRIPTION
**Pull request by breilly2**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by breilly2** [comment]
In my unwell fatigue yesterday afternoon, I accidentally merged this PR when I meant to merge the related firecloud-develop PR instead. @jacmrob, @ahaessly, please review this anyway to make sure there aren't any problems that should be retroactively fixed. Thanks.

File: automation/docker/application.conf.ctmpl [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{define "USER_DOMAIN"}}{{with $env := env "ENVIRONMENT"}}{{if eq $env "dev"}}test.firecloud.org{{else if eq $env "qa"}}quality.firecloud.org{{else}}firecloud.org{{end}}{{end}}{{end}}
 {{with $environment := env "ENVIRONMENT"}}
```
Comment (ahaessly) [inline automation/docker/application.conf.ctmpl:2]: we could move this with up to the top and then we probably would not need it to be repeated when setting USER_DOMAIN

Comment (breilly2) [reply automation/docker/application.conf.ctmpl:2]: Unfortunately not. `{{define ...}}` blocks must be at the very beginning of the template.

Comment (breilly2) [reply automation/docker/application.conf.ctmpl:2]: Except for comments, apparently: https://github.com/broadinstitute/firecloud-ui/pull/956/files#diff-d4032c2c63d9ef5fc3e8ef534f7e7dbf
Or maybe they just can't live inside of other template directives like `{{with ...}}`. I tried it and it doesn't work, and also remember reading something about it.

**Review by ahaessly — COMMENTED** [review]
👍

**Pull request merged**